### ISSUE TITLE
Clarification

### ISSUE THREAD

**Issue by koriavinash1** [rwightman/efficientdet-pytorch#75]
Hi @rwightman, 

Thanks for this wonderful work. I've few queries regarding preprocessing and outputs:

+ In the preprocessing step ToTensor or ToNumpy routines doesn't involve min-max normalization, is it the heuristic you followed, or does it have any reason? 
+ The output detections are not bounded, it takes both negative and positive values. Is there a reason for keeping it unbounded?
+ In some of my experiments with effdet-d0, resulted in NaN loss, not sure about the reason

Thank you,
Avinash

**Comment by ofekp** [comment]
something like this?

```
Loss is nan, stopping training
{'loss_box': tensor(nan, device='cuda:0', grad_fn=<AddBackward0>), 'loss_mask': tensor(0.7749, device='cuda:0', grad_fn=<BinaryCrossEntropyWithLogitsBackward>), 'loss_objectness': tensor(1.3523, device='cuda:0', grad_fn=<BinaryCrossEntropyWithLogitsBackward>), 'loss_rpn_box_reg': tensor(0.2977, device='cuda:0', grad_fn=<DivBackward0>)}
An exception has occurred, use %tb to see the full traceback.

SystemExit: 1
```

**Comment by Asifzm** [comment]
Hi, i have also encountered the [nan] valued loss with d0.
Looking for any configuration tips to avoid this issue :)

**Comment by Ekta246** [comment]
> Hi, i have also encountered the [nan] valued loss with d0.
> Looking for any configuration tips to avoid this issue :)

Hi, 
I change the learning rate to a lower learning rate and the issue is solved.
I know its just a getaway.

Thanks

**Comment by rwightman** [comment]
Focal loss is generally not the most stable. Other retinanet impl have similar concerns and the official EfficientDet is also sensitive to the learning rate, batch size, and optimizer settings. Overall normalization and pre/post processing are fairly similar to other implementations of same or similar models.
